cmake/toolchain/zephyr/0.11/target.cmake

```diff
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: Apache-2.0
+
+set(CROSS_COMPILE_TARGET_arm         arm-zephyr-eabi)
```

Comment:
We need to do something like below to get AArch64 target working by the way:
```
if(CONFIG_64BIT)
  set(CROSS_COMPILE_TARGET_arm  aarch64-zephyr-elf)
else()
  set(CROSS_COMPILE_TARGET_arm  arm-zephyr-eabi)
endif()
```

Reply:
Should be added in the aarch64 PR.

Reply:
cc @carlocaione

Reply:
ACK